src/framework/enhancement/css/ProgressiveEnhancement.css

```diff
@@ -1,4 +1,8 @@
 /* Progressive Enhancement: JS will reverse the display setup if it is enabled */
-.fl-progEnhance-enhanced {display:none}
+.fl-progEnhance-enhanced {
+    display: none;
+}
 
-.fl-progEnhance-basic {}
+.fl-progEnhance-basic {
+    /* Is this required? */
+}
```

Comment:
I think we should probably remove this. @amb26 and @colinbdclark do you recall if there was a specific reason for the empty declaration?